theano/sandbox/cuda/nvcc_compiler.py

```diff
@@ -260,8 +260,10 @@ def compile_str(
         preargs2 = []
         for pa in preargs:
             if pa.startswith('-Wl,'):
-                preargs1.append('-Xlinker')
-                preargs1.append(pa[4:])
+                # the -rpath option is not understood by the Microsoft linker
+                if sys.platform != 'win32':
```

Comment:
This condition remove all -Wl parameter. Can you be more specific like "if sys.platfrom != win32 or not ps.startswith('-Wl,-rpath')"